src/runtime/pay-flow.js

```diff
@@ -71,21 +77,30 @@ export class PayStartFlow {
    * @return {!Promise}
    */
   start() {
+    // Map the proration mode to the enum value (if proration exists).
+    const prorationMode = this.subscriptionRequest_.replaceSkuProrationMode;
+    if (prorationMode) {
+      this.subscriptionRequest_.replaceSkuProrationMode =
```

Comment:
We need to update this directly in the payment request, not the input structure. Because we use it for analytics, etc - it's not good if we update it inline and give API users back a different object than the one they provided.

Reply:
done